build/_k-standard-goals.shade

```diff
@@ -96,7 +96,7 @@ default DNX_TOOLS_FEED = 'https://www.myget.org/F/dnxtools/api/v2'
   @{
     if (IsBuildV2) 
     {
-        const string buildPattern = "src/**";
+        const string buildPattern = "src/**;test/**;samples/**";
```

Comment:
The way we do publishing \ Coherence builds now is we grab everything in the artifacts directory. Now that we'll have tests and samples copied to artifacts (https://github.com/aspnet/Universe/pull/258/files#diff-4405a98bc7f6daec30b9a687f626cbd4R115) is there a way to filter them out of being published to the feed? Should we change the structure of packages so `src` -> `artifacts/build`, `test` -> `artifacts/test`, `samples` -> `artifacts/samples`?